src/main/java/edu/kit/compiler/codegen/BasicBlocks.java

```diff
@@ -54,17 +73,20 @@ public String toString() {
 
         @Getter
         private final Block firmBlock;
+        @Getter
+        private final int blockId;
```

Comment:
Calling this `blockId` might be a bit misleading, as it is not the value used to access the entry in `BasicBlocks`. I think something like `jumpLabel` might be a better fit. Please correct me if I have misunderstood what the field is used for.

Reply:
Fair

Reply:
The `getLabel` and `getJumpLabel` methods are somewhat redundant now.